src/locale/locales/es/messages.po

```diff
@@ -8598,13 +8609,13 @@ msgstr "Alternar para habilitar o deshabilitar contenido para adultos"
 
 #: src/screens/Hashtag.tsx:84
 #: src/screens/Topic.tsx:71
-#: src/view/screens/Search/Search.tsx:499
+#: src/view/screens/Search/Search.tsx:500
 msgid "Top"
 msgstr "Top"
 
 #: src/Navigation.tsx:383
 msgid "Topic"
-msgstr ""
+msgstr "TemaS"
```

Comment:
Hello! There is a very small mistake: the 's' is capitalized

Reply:
It shouldn't even be there (Topic vs topics)